kafka-connect-cloud-common/src/main/scala/io/lenses/streamreactor/connect/cloud/common/config/kcqlprops/PropsKeyEnum.scala

```diff
@@ -37,6 +37,8 @@ object PropsKeyEnum extends Enum[PropsKeyEntry] {
   case object ReadStartLine extends PropsKeyEntry("read.text.start.line")
   case object ReadEndLine   extends PropsKeyEntry("read.text.end.line")
 
+  case object ReadLastEndLineMissing extends PropsKeyEntry("read.text.last.end.line.missing")
```

Comment:
MINOR: Could be named `ReadDespiteLastEndLineMissing` for more clarity (IMO), but it's not a blocker IMO

Reply:
I don't see how it's clear. 

* Read is the prefix for text related source functionality
* LastEndLineMissing is a flag (true/false)